_proposals/pup-0001.rst

```diff
@@ -0,0 +1,317 @@
+  the appropriate place.
+
+Pre-Proposal
+^^^^^^^^^^^^
+
+Discussing an idea via the appropriate mailing list prior to writing a formal proposal is
+encouraged. This is intended to save the author time by gathering feedback quickly before writing a
+concrete, detailed proposal.
+
+
+Authoring
+^^^^^^^^^
+
+One or more authors create the proposal as a pull request against
+[the pulpproject.org repo](http://github.com/pulp/pulpproject.org/) using the next sequential
+proposal number. The proposal is a single file, e.g. `pup-0022.rst`. A core dev is expected to
+merge any proposal with adequate clarity and in the expected format.
+
+
+Announcement
+^^^^^^^^^^^^
+
+Once a proposal is merged to the proposals area, an announcement would be sent to the appropriate
+list, which would include a very brief introduction and a link to the proposal in its web viewable
+form.
+
+
+Discussion and Feedback
+^^^^^^^^^^^^^^^^^^^^^^^
+
+General discussion would occur on the announcement thread on the appropriate mailing list. Threaded
+e-mail readers should make this conversation format easy to follow, while keeping the proposal
+document in-tact and easy to read.
+
+
+Revision
+^^^^^^^^
+
+Revision would occur as additional pull requests against the active proposal. This allows the
+changelog of a proposal to always be maintained. This is how Python does it. This also allows
+edits/updates to come with a commit message indicating the concerns addressed or improvements
+including links to responses via the archive. Collaboration is also easy with a collaborator sending
+revisions to the author as a patch or a new pull request against the proposal.
+
+
+Voting
+^^^^^^
+
+The +1, +0, -0, -1 feedback process is used, with votes sent as responses via the
+announcement/discussion mailing list. The voting has the following meanings:
+
+* +1: "I love the idea and I’m strongly committed to it."
+* +0: "Sounds OK to me."
+* -0: "I’m not thrilled, but I won’t stand in the way."
+* -1: "I strongly disagree and would be very unhappy to see the idea turn into reality." This type
+  of feedback must come with a statement of objective concerns explaining what harm will come from
+  adopting the proposal. Also, -1 feedback should not be used to express subjective preferences.
+
+Note that votes can come from anyone, not just core devs. This is directly modeled after [the
+Django process](https://docs.djangoproject.com/en/dev/internals/contributing/bugs-and-features/#how-we-make-decisions)
+who modeled their process after the Python and Apache communities.
+
+
+Deciding
+^^^^^^^^
+
+* If an obvious consensus arises via voting, the proposal is ratified. This is a bit vague, but it
+  works for the Django community.
+* Discussion on the thread can confirm if consensus is obvious or not.
+* Votes come from anyone, not just core devs
+* A -1 from a core dev is considered a veto vote and will block ratification until the vote is
```

Comment:
Should we define a "core dev" somewhere in this doc?

If so, maybe that would be defined as someone with commit permission to the Pulp source?